src/registrar/templates/home.html

```diff
@@ -121,22 +142,56 @@ <h2>Domain requests</h2>
+            {% if application.status == "started" or application.status == "action needed" or application.status == "withdrawn"  %}
               <a href="{% url 'edit-application' application.pk %}">
               <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24">
                 <use xlink:href="{%static 'img/sprite.svg'%}#edit"></use>
               </svg>
               Edit <span class="usa-sr-only">{{ application.requested_domain.name|default:"New domain request" }} </span>
 
-              {% else %}
+            {% else %}
               <a href="{% url 'application-status' application.pk %}">
               <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24">
                 <use xlink:href="{%static 'img/sprite.svg'%}#settings"></use>
               </svg>
               Manage <span class="usa-sr-only">{{application.requested_domain.name}} </span>
-              {% endif %}
+            {% endif %}
             </a>
           </td>
+          {% if has_deletable_applications %}
+          <td class="tablet-margin-top-2">
+            {% if application.status == "started" or application.status == "withdrawn"  %}
+              <a
+                id="button-toggle-delete-domain-alert-{{ forloop.counter }}"
+                href="#toggle-delete-domain-alert-{{ forloop.counter }}"
+                class="usa-button--unstyled"
+                aria-controls="toggle-delete-domain-alert-{{ forloop.counter }}"
+                data-open-modal
+              >
+                <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24">
+                  <use xlink:href="{%static 'img/sprite.svg'%}#delete"></use>
+                </svg>
+                Delete
+              </a>
+
+              <div
+                class="usa-modal wider-modal"
+                id="toggle-delete-domain-alert-{{ forloop.counter }}"
+                aria-labelledby="Are you sure you want to continue?"
+                aria-describedby="User will be removed"
+                data-force-action
+              >
+                <form method="POST" action="{% url "application-delete" pk=application.id %}">
+                  {% if application.requested_domain %}
+                    {% include 'includes/modal.html' with modal_heading="Are you sure you want to delete "|add:application.requested_domain.name|add:"?"  modal_description="This will remove the domain request from the .gov registrar. This action cannot be undone." modal_button=modal_button|safe %}
+                  {% else %}
+                    {% include 'includes/modal.html' with modal_heading="Are you sure you want to delete your domain request?" modal_description="This will remove the domain request from the .gov registrar. This action cannot be undone." modal_button=modal_button|safe %}
```

Comment:
@zandercymatics is this the language that'd show when the domain request has no domain name on it yet. Like in the case of a request name being "New domain request"?

Reply:
@gabydisarli Correct! You can test this yourself by starting a new domain request, then leaving midway through. Rather than what it is now (`Are you sure you want to delete your domain request?`) Would you like this to conform to the previous pattern defined in the figma? I.e.  `Are you sure you want to delete New domain request`? Or something like this with corrected grammar?

Reply:
@zandercymatics yes let's follow that pattern. 

Are you sure you want to delete `New domain request #`?

Reply:
@zandercymatics with the numbering of unnamed requests. Let's set the 1st one to be called "New domain request 1" instead of "New domain request, New domain request 2, ..."